kotlinlib/src/mill/kotlinlib/ktlint/KtlintModule.scala

```diff
@@ -32,14 +32,14 @@ trait KtlintModule extends JavaModule {
    * Ktlint configuration file.
    */
   def ktlintConfig: T[Option[PathRef]] = Task {
-    None
+    Some(PathRef(millSourcePath / ".editorconfig"))
   }
 
```

Comment:
How about defaulting to `Task.workspace / ".editorconfig"`? I'd guess most folks would have the config file at the root of their project (which is what Task.workspace is) rather than in each module's subfolder

Reply:
This should be an `Task.Input`